packages/jest-react/src/internalAct.js

```diff
@@ -102,6 +102,11 @@ export function act<T>(scope: () => Thenable<T> | T): Thenable<T> {
         // migrate existing tests.
         let didFlushWork;
         do {
+          // Flush scheduled rAF.
+          if (global.flushRequestAnimationFrameQueue) {
```

Comment:
I hacked in this global since `jsdom` doesn't have a way to flush `requestAnimationFrame` that I can tell (e.g. flushing timers doesn't work). @acdlite I'm curious to hear what the preferred solution for this is though.

Reply:
`requestAnimationFrame` should be flushed every 16ms starting with with legacy fake timers in Jest 27.0.5: https://github.com/facebook/jest/pull/11567. But React is still on Jest 26 (see https://github.com/facebook/react/pull/21575) and using legacy fake timers. 

So this might not be needed either if we land https://github.com/facebook/react/pull/21575 (which is blocked by https://github.com/facebook/jest/issues/11561) or switch to [modern fake timers](https://jestjs.io/docs/26.x/configuration#timers-string).

Reply:
Good to know, I'll add a comment for now.